packages/core/src/badge/Badge.tsx

```diff
@@ -10,7 +10,7 @@ export interface BadgeProps extends HTMLAttributes<HTMLSpanElement> {
   /**
    * The number to display on the badge
    */
-  value: number | string;
+  value?: number | string;
```

Comment:
what happens to the current badge if you don’t provide a value ? By changing the API like this, it is now a possibility.

Reply:
badge will be treated as a DOT badge without the value